x/auth/client/rest/decode.go

```diff
@@ -0,0 +1,101 @@
+package rest
+
+import (
+	"encoding/base64"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/go-amino"
+	"io/ioutil"
+	"net/http"
```

Comment:
Be sure imports are sorted correctly too (goimports) :)

Reply:
> Be sure imports are sorted correctly too (goimports) :)

Sorry, I reset my os and forgot install plugins.